packages/core/core/src/factories/useUserShippingFactory.ts

```diff
@@ -56,7 +56,7 @@ export const useUserShippingFactory = <USER_SHIPPING, USER_SHIPPING_ITEM>(
     }, 'useUserShipping-error');
 
     const addAddress = async ({ address, customQuery }) => {
-      Logger.debug('useUserShipping.addAddress', mask(address));
```

Comment:
why? we supposed to have even more calls like this due to security/data leak reasons